frontend/server/utils.test.ts

```
@@ -36,4 +36,26 @@ describe('utils', () => {
       dst.once('readable', () => expect(dst.read().toString()).toBe(input));
     });
   });
+
+  describe('pruneAndConcatPath', () => {
+    it('undefined prefixPrunePath', () => {
+      const path = pruneAndConcatPath('/data', 'a/b/c', undefined);
+      expect(path).toEqual('/data/a/b/c');
+    });
+
+    it('with prefixPrunePath', () => {
+      const path = pruneAndConcatPath('/data', 'a/b/c', 'a');
+      expect(path).toEqual('/data/b/c');
+    });
+
+    it('with multiple layer prefixPrunePath', () => {
+      const path = pruneAndConcatPath('/data', 'a/b/c', 'a/b');
+      expect(path).toEqual('/data/c');
+    });
+
+    it('with not exist prefixPrunePath', () => {
+      const path = pruneAndConcatPath('/data', 'a/b/c', 'other');
+      expect(path).toEqual('/data/a/b/c');
```

Shall we throw an error here?